imagesgallery/imagesgallery.py

```
@@ -237,7 +256,15 @@ def remove_files(self, data, suffix=''):  # pylint: disable=unused-argument
             from cms.djangoapps.contentstore.asset_storage_handler import delete_asset  # pylint: disable=import-outside-toplevel
         except ImportError:
             from cms.djangoapps.contentstore.views.assets import delete_asset  # pylint: disable=import-outside-toplevel
-        delete_asset(self.course_id, asset_key)
+        try:
+            delete_asset(self.course_id, asset_key)
+        except Exception as e:  # pylint: disable=broad-except
+            log.exception(e)
```

Will you want to raise an exception here?

---

I think what I could do is catch the exception I'm the most worried about when the course asset doesn't exist but is still part of the contents list. I'll do that and let you know!

---

Changes done!